[2025-10-29T11:56:56.685+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-29T11:55:00+00:00 [queued]>
[2025-10-29T11:56:56.693+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-29T11:55:00+00:00 [queued]>
[2025-10-29T11:56:56.694+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-10-29T11:56:56.705+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): ConsumKafka> on 2025-10-29 11:55:00+00:00
[2025-10-29T11:56:56.713+0000] {standard_task_runner.py:60} INFO - Started process 174 to run task
[2025-10-29T11:56:56.717+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag1_calcul_trajet_kafka', 'ConsumKafka', 'scheduled__2025-10-29T11:55:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/dag1_kafka_processing.py', '--cfg-path', '/tmp/tmpmfdomxhk']
[2025-10-29T11:56:56.720+0000] {standard_task_runner.py:88} INFO - Job 2: Subtask ConsumKafka
[2025-10-29T11:56:56.798+0000] {task_command.py:423} INFO - Running <TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-29T11:55:00+00:00 [running]> on host b446a8114f7f
[2025-10-29T11:56:56.886+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag1_calcul_trajet_kafka' AIRFLOW_CTX_TASK_ID='ConsumKafka' AIRFLOW_CTX_EXECUTION_DATE='2025-10-29T11:55:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-29T11:55:00+00:00'
[2025-10-29T11:56:56.891+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('172.18.0.3', 9092)]>: connecting to kafka:9092 [('172.18.0.3', 9092) IPv4]
[2025-10-29T11:56:56.898+0000] {conn.py:618} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <checking_api_versions_recv> [IPv4 ('172.18.0.3', 9092)]>: Broker version identified as 2.6
[2025-10-29T11:56:56.899+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.3', 9092)]>: Connection complete.
[2025-10-29T11:56:56.900+0000] {subscription_state.py:181} INFO - Updating subscribed topics to: ('source',)
[2025-10-29T11:56:56.901+0000] {dag1_kafka_processing.py:49} INFO - ‚úÖ Consommateur Kafka cr√©√© pour le topic 'source'. En attente de messages...
[2025-10-29T11:56:56.904+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.3', 9092)]>: connecting to kafka:9092 [('172.18.0.3', 9092) IPv4]
[2025-10-29T11:56:56.905+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.3', 9092)]>: Connection complete.
[2025-10-29T11:56:56.906+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.3', 9092)]>: Closing connection. 
[2025-10-29T11:56:57.010+0000] {cluster.py:401} INFO - Coordinator for group/***-consumers-dag1 is ('coordinator-1', 'kafka', 9092, None)
[2025-10-29T11:56:57.011+0000] {base.py:788} INFO - Discovered coordinator coordinator-1 for group ***-consumers-dag1
[2025-10-29T11:56:57.012+0000] {base.py:848} INFO - Starting new heartbeat thread
[2025-10-29T11:56:57.013+0000] {consumer.py:365} INFO - Revoking previously assigned partitions () for group ***-consumers-dag1
[2025-10-29T11:56:57.014+0000] {base.py:419} INFO - Failed to join group ***-consumers-dag1: NodeNotReadyError: coordinator-1
[2025-10-29T11:56:57.015+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.3', 9092)]>: connecting to kafka:9092 [('172.18.0.3', 9092) IPv4]
[2025-10-29T11:56:57.016+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connected> [IPv4 ('172.18.0.3', 9092)]>: Connection complete.
[2025-10-29T11:56:57.116+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-29T11:56:57.120+0000] {base.py:637} INFO - Received member id kafka-python-2.2.15-efdf195d-64ee-4f06-b1e8-6e32b647ac7d for group ***-consumers-dag1; will retry join-group
[2025-10-29T11:56:57.121+0000] {base.py:419} INFO - Failed to join group ***-consumers-dag1: [Error 79] MemberIdRequiredError
[2025-10-29T11:56:57.121+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-29T11:57:01.902+0000] {dag1_kafka_processing.py:64} INFO - ‚è≥ Aucun message encore... nouvelle tentative.
[2025-10-29T11:57:06.128+0000] {base.py:594} INFO - Successfully joined group ***-consumers-dag1 <Generation 1 (member_id: kafka-python-2.2.15-efdf195d-64ee-4f06-b1e8-6e32b647ac7d, protocol: range)>
[2025-10-29T11:57:06.129+0000] {base.py:596} INFO - Elected group leader -- performing partition assignments using range
[2025-10-29T11:57:06.134+0000] {subscription_state.py:254} INFO - Updated partition assignment: []
[2025-10-29T11:57:06.135+0000] {consumer.py:249} INFO - Setting newly assigned partitions () for group ***-consumers-dag1
[2025-10-29T11:57:08.905+0000] {dag1_kafka_processing.py:64} INFO - ‚è≥ Aucun message encore... nouvelle tentative.
[2025-10-29T11:57:10.912+0000] {base.py:951} WARNING - Heartbeat failed for group ***-consumers-dag1 because it is rebalancing
[2025-10-29T11:57:10.913+0000] {consumer.py:365} INFO - Revoking previously assigned partitions () for group ***-consumers-dag1
[2025-10-29T11:57:10.913+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-29T11:57:10.917+0000] {base.py:594} INFO - Successfully joined group ***-consumers-dag1 <Generation 2 (member_id: kafka-python-2.2.15-efdf195d-64ee-4f06-b1e8-6e32b647ac7d, protocol: range)>
[2025-10-29T11:57:10.917+0000] {base.py:596} INFO - Elected group leader -- performing partition assignments using range
[2025-10-29T11:57:10.921+0000] {subscription_state.py:254} INFO - Updated partition assignment: []
[2025-10-29T11:57:10.921+0000] {consumer.py:249} INFO - Setting newly assigned partitions () for group ***-consumers-dag1
[2025-10-29T11:57:13.925+0000] {base.py:951} WARNING - Heartbeat failed for group ***-consumers-dag1 because it is rebalancing
[2025-10-29T11:57:13.926+0000] {consumer.py:365} INFO - Revoking previously assigned partitions () for group ***-consumers-dag1
[2025-10-29T11:57:13.927+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-29T11:57:13.929+0000] {base.py:594} INFO - Successfully joined group ***-consumers-dag1 <Generation 3 (member_id: kafka-python-2.2.15-efdf195d-64ee-4f06-b1e8-6e32b647ac7d, protocol: range)>
[2025-10-29T11:57:13.930+0000] {base.py:596} INFO - Elected group leader -- performing partition assignments using range
[2025-10-29T11:57:13.932+0000] {subscription_state.py:254} INFO - Updated partition assignment: [('source', 0)]
[2025-10-29T11:57:13.933+0000] {consumer.py:249} INFO - Setting newly assigned partitions (('source', 0),) for group ***-consumers-dag1
[2025-10-29T11:57:13.936+0000] {dag1_kafka_processing.py:60} INFO - üì© Message re√ßu : {'confort': 'medium', 'prix_base_per_km': 2, 'properties-client': {'logitude': 2.304219350991466, 'latitude': 48.848356057452975, 'nomclient': 'FALL', 'telephoneClient': '060786575'}, 'properties-driver': {'logitude': 3.7047475272399, 'latitude': 40.41941392691913, 'nomDriver': 'DIOP', 'telephoneDriver': '0760786575'}}
[2025-10-29T11:57:13.940+0000] {base.py:1052} INFO - Stopping heartbeat thread
[2025-10-29T11:57:13.941+0000] {base.py:891} INFO - Leaving consumer group (***-consumers-dag1).
[2025-10-29T11:57:13.945+0000] {base.py:906} INFO - LeaveGroup request for group ***-consumers-dag1 returned successfully
[2025-10-29T11:57:13.946+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.3', 9092)]>: Closing connection. 
[2025-10-29T11:57:13.947+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connected> [IPv4 ('172.18.0.3', 9092)]>: Closing connection. 
[2025-10-29T11:57:13.947+0000] {python.py:201} INFO - Done. Returned value was: {'confort': 'medium', 'prix_base_per_km': 2, 'properties-client': {'logitude': 2.304219350991466, 'latitude': 48.848356057452975, 'nomclient': 'FALL', 'telephoneClient': '060786575'}, 'properties-driver': {'logitude': 3.7047475272399, 'latitude': 40.41941392691913, 'nomDriver': 'DIOP', 'telephoneDriver': '0760786575'}}
[2025-10-29T11:57:13.980+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag1_calcul_trajet_kafka, task_id=ConsumKafka, execution_date=20251029T115500, start_date=20251029T115656, end_date=20251029T115713
[2025-10-29T11:57:14.026+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-29T11:57:14.050+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
